main.js

```diff
@@ -50,6 +52,44 @@ function createWindow(commandLineArguments, workingDirectory) {
 
     // Create the browser window.
     let mainWindow = new BrowserWindow({ width: 800, height: 600, icon: path.join(__dirname, "images", "Oni_128.png") })
+    let menu = defaultMenu(app, shell);
+
+  menu.unshift ({
+    label: 'Oni',
+    submenu: [
+      {
+        label: 'Quit',
+        click: (item, focusedWindow) => {
+          app.quit()
+        }
+      }
+    ]
+  })
+
+//    deleting 'Toggle Developer Tools'
+   menu[2].submenu.splice(2,1)
+   menu[4].submenu = [
+       {
+        label: 'Learn more',
+        click: (item, focusedWindow) => {
+        shell.openExternal('https://github.com/extr0py/oni#introduction');
+        }
+      },
+        {
+        label: 'Issues',
```

Comment:
Thanks for adding these options, that's awesome! This will give us a solid foundation to add more menu items down the road as well.